main.tf

```diff
@@ -2,7 +2,7 @@ locals {
   # Atlantis
   atlantis_url = "https://${try(coalesce(
     try(var.atlantis.fqdn, null),
```

Comment:
hmm, ok. well we can at least save one `try()` for now with the following. I believe this should work if what you are proposing works:
```hcl
atlantis_url = "https://${try(coalesce(
    try(var.atlantis.fqdn, module.alb.route53_records["A"].fqdn, null),
    module.alb.dns_name,
  ), "")}"
```

Reply:
That's a good point. Just updated the commit - seems to work!